components/hab/src/command/studio/enter.rs

```diff
@@ -86,6 +86,23 @@ mod inner {
                     let ident = PackageIdent::from_str(&format!("{}/{}",
                                                                 super::STUDIO_PACKAGE_IDENT,
                                                                 version[0]))?;
+                    // TODO(SM): This is the minimum change needed to allow the studio to specify
+                    // its dependencies. This is a duplicate of the code in `hab pkg exec` and
+                    // should be refactored.
```

Comment:
Rather than a `TODO`, let's file an issue with a link back here so it's discoverable

Reply:
Or just refactor now; it doesn't look like it would be too disruptive. What am I missing that makes it so?

Reply:
Refactoring this would involve updating code in `hab pkg exec` and `core/os/process` or `PackageInstall` which I feel should be a separate PR. Additionally, when I started looking at making the same change for the Windows studio, I found a lot of duplication and not very easy to grok logic to determine how we enter a studio. Refactoring that will be a larger slice of work that I again feel it should be a separate PR. 

Given that this change is to help unblock some changes we're making to our release process and it's updating a component that is both critical and not as well tested as we'd like, I'd like to keep this change as minimal as possible to make discovery of any potential issues easy to debug.

I've filed #6633 and #6634 to track the changes needed.

Reply:
Ok, can we replace the TODO with references to those issues?